showkase/src/main/java/com/airbnb/android/showkase/ui/ShowkaseBrowserApp.kt

```diff
@@ -370,10 +370,74 @@ internal fun ShowkaseBodyContent(
     groupedTypographyMap: Map<String, List<ShowkaseBrowserTypography>>,
     showkaseBrowserScreenMetadata: MutableState<ShowkaseBrowserScreenMetadata>
 ) {
+    val isOnlyComponents = groupedColorsMap.values.isEmpty() && groupedTypographyMap.values.isEmpty()
     NavHost(
         navController = navController,
-        startDestination = ShowkaseCurrentScreen.SHOWKASE_CATEGORIES.name
-    ) {
+        startDestination = if (isOnlyComponents) {
```

Comment:
I think we should make this generic i.e directly jump to the respective groups screen if only one type of elements are present.

For example, if there are only colors, it should directly go to the colors groups screen. Same with typography or any other element that we might add support for in the future.

Reply:
ok, it makes sense, I can change this to be the default behaviour